launchapp.py

```diff
@@ -9,8 +9,14 @@
 import urllib2
 import webbrowser
 import uuid
+import socket
 
 source_exec = sys.argv[1]
+host_ip = socket.gethostbyname(socket.gethostname())
+try:
+    vm_ip = sys.argv[4]
+except:
```

Comment:
It is not correct Python practice to catch bare exceptions.  Please specify which exception you want to catch here.